js/samples/04.ai-apps/i.teamsChefBot-streaming/src/index.ts

```diff
@@ -104,11 +106,34 @@ const prompts = new PromptManager({
     promptsFolder: path.join(__dirname, '../src/prompts')
 });
 
+const endStreamHandler: PromptCompletionModelResponseReceivedEvent = (ctx, memory, response, streamer) => {
+    // Ignore events for other contexts
+    if (!streamer) {
```

Comment:
Are there currently other events that might be caught here or is this a guard for the future?

Reply:
for the future